### ISSUE TITLE
Is it possible to draw on Apple Vision Pro through bevy and composerLayer?

### ISSUE THREAD

**Issue by random-yang** [bevyengine/bevy#16276]
I raise this question more to trigger some discussions. The ideal solution would enable developers to draw and render content on Apple Vision Pro using Bevy via composerLayer. This solution would establish the necessary bridge or bindings between Vision Pro’s rendering layers and Bevy, allowing developers to implement interactive 3D content using Bevy's engine structure and Rust's language capabilities. This could involve direct support within Bevy for Vision Pro or a separate plugin/module that handles integration with composerLayer.

**Comment by BenjaminBrienen** [comment]
This might belong in a Discussions post instead. Unfortunately, I don't know the answer to your question and not many people are using bevy for VR, let alone for the Vision Pro. Maybe someone else can chime in.

**Comment by random-yang** [comment]
> This might belong in a Discussions post instead. Unfortunately, I don't know the answer to your question and not many people are using bevy for VR, let alone for the Vision Pro. Maybe someone else can chime in.

Yes, I just want to explore the possibility of reusing Rust's rendering capabilities in AVP development in the future.
